plugins/outputs/quix/quix.go

```diff
@@ -0,0 +1,130 @@
+// quix.go
+package quix
+
+import (
+	"crypto/sha256"
+	"strings"
+
```

Comment:
You need to embed the `sample.conf` file...
```suggestion
	"crypto/sha256"
	_ "embed"
	"strings"

```